platforms.c

```diff
@@ -474,6 +474,16 @@ static int __start_pvlogger_for_platform(struct pv_platform *platform,
 	if (!pid) {
 		char namespace[64];
 		int ns_fd = -1;
+
+		if (getuid() == 0) {
```

Comment:
trying to undertand this. What are the getuid around doing?

Reply:
@anibalportero only for containers that get started as root we do this... if we are not root we cannot adjust oom score and it woudl fail.